backend/sftp/sftp.go

```diff
@@ -286,13 +287,33 @@ func (r *SFTP) Create() (backend.Blob, error) {
 	return &blob, nil
 }
 
+func Join(parts ...string) string {
```

Comment:
This function's body can be replaced by calling `path.Clean(path.Join(parts...))`.